content/articles/Integrating-mongodb-with -java/index.md

```diff
@@ -0,0 +1,241 @@
+
+![Deleted Data](/engineering-education/integrating-mongodb-with-java/delete-data.png)
+
+### Delete the collection
+ To delete a collection that has been created the code below is used.
+ ``` java
+ //it deletes the whole collection
+  collection.drop();
+ ```
+ ### Eclipse versus Intellij Idea in MongoDB integration with java
+The CRUD operations are performed the same way in IntelliJ and Eclipse, but connecting MongoDB with Java in Eclipse is different. The IntelliJ idea makes use of dependencies, which are Jason files in Javascript. Here, the Java driver is included in the dependencies.
+Dependencies used should be added to the XML file. Mongodb-driver requires the following dependencies: bson and mongodb-driver-core as shown below. 
+```java      
+<dependencies>
+    <dependency>
+        <groupId>org.mongodb</groupId>
+        <artifactId>mongo-java-driver</artifactId>
+        <version>3.12.10</version>//the latestmongo java driver version
+    </dependency>
+</dependencies>
+```
+  
+Create a new class (SRC-main-Java-new class) and import the following packages
+```java
+import com.mongodb.*; 
+import java.net.UnknownHostException;
+
+```
+Inside the public class created, a client should be initialized to run the project with the mongo client syntax. You can decide to declare your client's collections and databases as static in the main file as shown below.
+``` java
+import com.MongoDB.*; 
+import java.net.UnknownHostException;
+public class mongoDB {
+public static MongoClient mongoClient;
+public static DB database;
+public static DBCollection collectiontest
+public static void main(string[]args) throws UnknownHostExcepton {mongoClient= new MongoClient(new MongoClientURL("mongodb://localhost:27017"));          
+}
+}
+```
+In the main class, you'll start the mongo client using the following code snippet.
+  
+```java
+mongoClient= new MongoClient(new MongoClientURL("mongodb://localhost:27017"));
+```
+To instantiate the database add the following code.
+```java
+database= mongoClient.getDB(dbname: "SchoolManagementSystem);
+```
+To create a collection you use
+```java
+collectiontest=database.getCollection(name:"student");
+```
+
+[More on connecting MongoDB with Java using ecllipse](https://www.jetbrains.com/help/idea/mongodb.html)
+
+### Conclusion
+Java and MongoDB are two of the most popular software development technologies, so it's no surprise that they complement each other well. Whether you're developing on a local or on-premises MongoDB installation or going cloud-first with MongoDB Atlas, the MongoDB Java Driver and surrounding ecosystem make integrating MongoDB into your development process a breeze. In this tutorial, we have looked at setting up a MongoDB server. We have also looked at how to establish a connection between MongoDB and Java and how to operate with different CRUD operations. I hope you found this article helpful.!
+ [This Github URL will be of help](https://github.com/dboissier/mongo4idea)

```

Comment:
This is not your project.

The codes that you used in the tutorial must be here.